### PR TITLE
Midiwizard: improve UX in controls menus

### PR DESCRIPTION
**Pull request by ronso0**
https://bugs.launchpad.net/mixxx/+bug/1901107

- [x] add separators in many places
- [x] move _Sampler 1-64_ controls into submenus
- [x] group **Libary** navigation controls into submenu
- [x] **Mixer**: create submenus _Crossfader /Orientation_, _Main_, _Headphone_
- [x] **Equalizers**: move to _Mixer_
- [x] **Transport**: move _Volume_, _Mute_ & _Pfl_ to Mixer
- [x] split _Speed / Pitch_ menu into **Speed / Tempo** and **Pitch (Musical Key)**
- [x] **Looping** & **Beat-Looping**: merge, sort controls into submenus _Loop_ & _Loop Roll_, remove redundant _Loopmove_ actions
- [x] **Effects**: _Unit N_ > _Effect Unit N_, move unit menus to main _Effects_ menu (remove _Effect Rack 1_ level)
- [x] **Effects**: unclutter Fx assign GUI strings

Anything else ?

### CODE REVIEW

**Comment by ronso0** [comment]
I don't post screenshots.
Please check the menu in 2.3 first, then try this branch: improvement should be obvious :)

**Comment by Holzhaus** [comment]
Weird, something is wrong with the hotcues, the last few of them have no labels. 
Screenshot from 2020-10-23 19-34-45

**Comment by ronso0** [comment]
I didn't touch that and I can see all labels up to n°37
Seems the menu is too tall, you need a bigger screen :)
Has this been working for you before in 2.3?

**Comment by ronso0** [comment]
And how does a sampler submenu with sampler 1-64 look for you?

**Comment by ronso0** [comment]
Anyway, we may put hotcues 17++ into a submenu `Hotcues 17-37` to relieve those estimated 99,3% Mixxxers who need 16 hotcues max.

File: src/controllers/controlpickermenu.cpp [hunk 1]
```diff
@@ -890,18 +1098,37 @@ void ControlPickerMenu::addPlayerControl(QString control, QString controlTitle,
...
-        // PlayerManager::groupForSampler is 0-indexed.
-        QString prefix = m_samplerStr.arg(i);
-        QString group = PlayerManager::groupForSampler(i - 1);
-        addSingleControl(group, control, controlTitle, controlDescription,
-                         controlMenu, prefix, prefix);
-
+    if (samplerControls) {
+        parented_ptr<QMenu> samplerControlMenu = make_parented<QMenu>(tr("Samplers"), controlMenu);
+        controlMenu->addMenu(samplerControlMenu);
+        parented_ptr<QMenu> samplerResetControlMenu = nullptr;
```
Comment (Holzhaus) [inline src/controllers/controlpickermenu.cpp:1104]: Does parented_ptr make sense if the variable is not stored?

Comment (ronso0) [reply src/controllers/controlpickermenu.cpp:1104]: probabyl not. I'll check.
I confess I just lazily copied that from the main menu above.

**Comment by ronso0** [comment]
> Anyway, we may put hotcues 17++ into a submenu `Hotcues 17-37` to relieve those estimated 99,3% Mixxxers who need 16 hotcues max.

otoh yet another submenu doesn't look nice.
I'll check if there's some expander like `...` or `More ...` available for QMenu.
but that shouldn't hold up this PR.

**Comment by ronso0** [comment]
Comments are addressed and I added the additional hotcues menu:
image
We could also name that menu "More hotcues ..." instead of the dynamic title.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Pull request merged**